charms/slurmctld/src/interface_influxdb.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
+"""Influxdb interface."""
+import json
```

Comment:
[nit]: I like to have newline in between the module docstring and the first important statement:

```suggestion
"""Influxdb interface."""

import json
```